src/main/java/com/spotinst/sdkjava/model/OceanK8sVirtalNodeGroupConverter.java

```diff
@@ -0,0 +1,505 @@
+package com.spotinst.sdkjava.model;
+
+import com.spotinst.sdkjava.enums.AwsVolumeTypeEnum;
+import com.spotinst.sdkjava.enums.K8sVngHttpTokensEnum;
+import com.spotinst.sdkjava.model.api.ocean.kubernetes.*;
+import com.spotinst.sdkjava.model.bl.ocean.kubernetes.*;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class OceanK8sVirtalNodeGroupConverter {
+    public static ApiK8sVirtualNodeGroup toDal(K8sVirtualNodeGroup launchSpec) {
```

Comment:
Use vng instead of launchspec to be consistent.

Reply:
Rename launchSpec to vngLaunchSpec